### ISSUE TITLE
Could Not Save Token Java Error

### ISSUE THREAD

**Issue by TactiCool-HUN** [RPTools/maptool#3576]
Short Description: when I pull in a .rptok token from the library, change something on it and then try to save it in the same folder and on the same name it throws an error.

Maptool version 1.11.5
OS: Windows 10

Steps to reproduce:
- Open maptool
- Drag & Drop a token (.rptok) from the maptool library
- Change something on it
- Right click save on the token
- Try to save it in the original folder and on the original name
- Click yes when it asks if you'd like to override it
- Done

Here is the error message I get:
![unknown](https://user-images.githubusercontent.com/108133191/187955925-9c7333b8-50cc-49a7-8dca-ba6cfefde0ed.png)
The end part is Hungarian, here is my translation of it: "The process does not have access to the file, because another process is using it."

The issues can be solved by saving the campaign file the changed token is in, closing maptool, then reopening it and saving the token the same way afterwards.

**Comment by FullBleed** [comment]
I've encountered a similar issue when working on images with Photoshop that I add to table in MT.

To reproduce:

1)  Create a Table called "Test" and add an image to a line.
2)  Go Photoshop, open the image, edit it, and save it.
3)  Go to MT, open the table and update the image you added to the new, edited image.
4)  Go back to Photoshop and make some more changes to the image and attempt to save it again:  
![in use](https://user-images.githubusercontent.com/37131742/187980313-f4d06b7d-3f72-4f91-bb25-cf92b26949dd.jpg)
5)  Close MapTool and you can save the file in Photoshop without a problem.

So, basically, it seems like MT is sometimes keeping files "in use/open" when it shouldn't be.
